test/dash_parser_segment_list_unit.js

```diff
@@ -267,5 +267,35 @@ describe('DashParser.SegmentList', function() {
       Dash.testSegmentIndex(done, source, references);
     });
   });
+
+  describe('Segment start', function() {
+    it('shoud be adjusted with presentationTimeOffset', function(done) {
+      var source = [
+        '<MPD>',
+        ' <Period id="0" start="PT0.000S">',
+        '  <AdaptationSet mimeType="audio/mp4" segmentAlignment="true">',
+        '   <Representation id="1" bandwidth="49000" codecs="mp4a.40.5">',
+        '    <BaseURL>http://example.com</BaseURL>',
+        '    <SegmentList presentationTimeOffset="120" duration="10"' +
+            ' startNumber="1">',
+        '     <SegmentURL media="s-120.m4a" />',
```

Comment:
You are right... I missed that, I saw them identical... :) I think I was tired. Thanks.